defaults/defaults.go

```diff
@@ -16,6 +16,9 @@ const (
 	AgentResourceQuota      = "cilium-resource-quota"
 	AgentImage              = "quay.io/cilium/cilium"
 
+	EnvoyDaemonSetName = "cilium-envoy"
+	EnvoyConfigMapName = "cilium-envoy-config"
```

Comment:
The similarity with the CiliumEnvoyConfig CRD is a bit unfortunate but not significant, IMO.

Reply:
agree, but context is king :) going to mark this as resolved - also because cilium/cilium PR already have been merged in the meantime.